database/schema.sql

```diff
@@ -1,88 +1,88 @@
+CREATE TABLE cancellations (
+    cancellation_id SERIAL PRIMARY KEY,
+    scheduled_arrival TIMESTAMP NOT NULL,
+    cancellation_type_id INT REFERENCES cancellation_types(cancellation_type_id),
+    station_id INT REFERENCES stations(station_id),
+    service_id INT REFERENCES services(service_id)
 );
 
-CREATE TABLE Arrivals(
-    ArrivalID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    ScheduledArrival timestamp NOT NULL,
-    ActualArrival timestamp NOT NULL,
-    StationID INT REFERENCES Stations(StationID),
-    ServiceID INT REFERENCES Services(ServiceID)
+CREATE TABLE arrivals (
+    arrival_id SERIAL PRIMARY KEY,
+    scheduled_arrival TIMESTAMP NOT NULL,
+    actual_arrival TIMESTAMP NOT NULL,
+    station_id INT REFERENCES stations(station_id),
+    service_id INT REFERENCES services(service_id)
 );
 
-CREATE TABLE Users(
-    UserID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    FirstName varchar(30) NOT NULL,
-    LastName varchar(60) NOT NULL,
-    PhoneNumber varchar(20),
-    Email varchar
+CREATE TABLE users (
+    user_id SERIAL PRIMARY KEY,
+    first_name VARCHAR(30) NOT NULL,
+    last_name VARCHAR(60) NOT NULL,
+    phone_number VARCHAR(20),
+    email VARCHAR
 );
 
-CREATE TABLE StationSubscriptions(
-    StationSubscriptionID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    IsActive BOOLEAN NOT NULL,
-    UserID INT REFERENCES Users(UserID),
-    StationID INT REFERENCES Stations(StationID)
+CREATE TABLE station_subscriptions (
+    station_subscription_id SERIAL PRIMARY KEY,
+    is_active BOOLEAN NOT NULL,
+    user_id INT REFERENCES users(user_id),
+    station_id INT REFERENCES stations(station_id)
 );
 
-CREATE TABLE OperatorSubscriptions(
-    OperatorSubscriptionID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    IsActive BOOLEAN NOT NULL,
-    UserID INT REFERENCES Users(UserID),
-    OperatorID INT REFERENCES Operators(OperatorID)
+CREATE TABLE operator_subscriptions (
+    operator_subscription_id SERIAL PRIMARY KEY,
+    is_active BOOLEAN NOT NULL,
+    user_id INT REFERENCES users(user_id),
+    operator_id INT REFERENCES operators(operator_id)
 );
 
-CREATE TABLE Incidents(
-    IncidentRecordID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    OperatorID INT REFERENCES Operators(OperatorID),
-    CreationDate timestamp NOT NULL, 
-    Description text,
-    Summary text, 
-    StartDate timestamp NOT NULL,
-    EndDate timestamp,
-    InfoLink text,
-    AffectedRoutes text,
-    Planned BOOLEAN NOT NULL,
-    IncidentNumber varchar(32) UNIQUE NOT NULL,
-    LastUpdated timestamp
+CREATE TABLE incidents (
+    incident_id SERIAL PRIMARY KEY,
+    operator_id INT REFERENCES operators(operator_id),
+    creation_date TIMESTAMP NOT NULL,
+    description TEXT,
+    summary TEXT,
+    start_date TIMESTAMP NOT NULL,
+    end_date TIMESTAMP,
+    info_link TEXT,
+    affected_routes TEXT,
+    planned BOOLEAN NOT NULL,
+    incident_uuid VARCHAR(32) UNIQUE NOT NULL,
+    last_updated TIMESTAMP
 );
```

Comment:
Everything looks good in terms of the naming convention switch.
Have you checked that it works successfully with the initialise_tables.sh bash script?

Reply:
I have yep.